packages/core/lib/configAdapter.js

```diff
@@ -38,7 +32,7 @@ function configureManagedGanache(config, networkConfig, mnemonic) {
 
   const network_id = getFirstDefinedValue(
     networkConfig.network_id,
-    Date.now() // Use current time as default
+    5777 // Use as default network_id
```

Comment:
Any particular reason for this change?

Reply:
I found out a different behavior with this. In `environment.js`, there is a logic to set the `network_id` to 5777 if it is undefined in the `config.network`. So when we use `Date.now()` as default, it conflicts with the set `network_id` 5777 and throws an error when we use the `migrate` command.

Reply:
Oh, interesting.  Makes sense to stick with 5777 then.